client/lib/products-values/index.js

```diff
@@ -252,6 +252,12 @@ export function isJetpackCloudProductSlug( productSlug ) {
 	);
 }
 
+export function isJetpackSearch( product ) {
```

Comment:
This function exists [here](https://github.com/Automattic/wp-calypso/blob/2f90a6236742e8ddf98cca75983a5a540034c88a/client/lib/products-values/constants.js#L46). This does seem like a better place for it, but the latter is in use in multiple places.